packages/js/tests/integrations-page/SiteKitIntegration.test.js

```diff
@@ -164,11 +168,52 @@ describe( "SiteKitIntegration", () => {
 		} );
 
 		expect( apiFetch ).toHaveBeenCalledWith( expect.objectContaining( {
-			url: urlsProps.consentManagementUrl,
+			url: props.consentManagementUrl,
 			method: "POST",
 			data: { consent: "false" },
 		} ) );
 		expect( disconnectDialog ).not.toBeInTheDocument();
 		expect( screen.getByRole( "button", { name: "Connect Site Kit by Google" } ) ).toBeInTheDocument();
 	} );
+
+	describe( "should show warning when user don't have permission to install plugins and should disable the link", () => {
```

Comment:
Again, this sentence does not flow due to missing words., please change below too